wiki/osu!_File_Formats/Osu_(file_format)/en.md

```diff
@@ -194,26 +194,58 @@ Hit Circle Syntax:
 `x,y,time,type,hitSound,addition`
 `164,260,2434,1,0,0:0:0:0:`
 
+x (Integer) ranges from 0 to 512 (inclusive) and y (Integer) ranges from 0 to 384 (inclusive).
+
+time (Integer) is in milliseconds from the beginning of the song.
+
+type (Integer) is a bitmap:
+
+1 = Circle, 2 = Slider, 4 = New combo, 8 = Spinner. More significant bits can be used to choose the next combo color.
+
+hitSound (Integer) is a bitmap:
+
+2 = Whistle, 4 = Finish, 8 = Clap
```

Comment:
I don't see 1 used anywhere and I'm not aware of any times when auto would play something besides normal. As far as I can tell, there is always a normal hitsound, with whistle/finish/clap added; the first bit doesn't do anything.